tools/cli/go-whisk-cli/commands/util.go

```diff
@@ -41,13 +41,7 @@ import (
     "bytes"
 )
 
-type QualifiedName struct {
-    namespace   string  // namespace. does not include leading '/'.  may be "" (i.e. default namespace)
-    packageName string  // package.  may be "".  does not include leading/trailing '/'
-    entity      string  // entity.  should not be ""
-    entityName  string  // pkg+entity
-}
-
+// TODO: DELETE THIS BLOCK OF CODE ONCE API-EXPERIMENTAL IS REMOVED!!!!!********
```

Comment:
Sorry - I wasn't sure about the timeline for api-experimental being removed, and so I was trying to be proactive in case there was some chance this might get merged before that specific change. I reverted all api-experimental code back to the way it was before I made changes to it, and a added this code back in so that tests would still run properly in playground/travis.